instance/models/openedx_instance.py

```diff
@@ -227,7 +227,10 @@ def appserver_is_shut_down(appserver):
             """
             if appserver.status == AppServerStatus.Terminated:
                 return True
-            configuration_failed = appserver.status == AppServerStatus.ConfigurationFailed
+            configuration_failed = (
+                appserver.status == AppServerStatus.ConfigurationFailed or
+                appserver.status == AppServerStatus.Error
+            )
             vm_terminated = appserver.server.status == ServerStatus.Terminated
             return configuration_failed and vm_terminated
```

Comment:
@itsjeyd Ah, that clears things up - in that case, this PR does seem to make the logic change we want. Can you take another look @pomegranited ?